wire-ios/Wire-iOS Tests/CreateSecureGuestLinkViewModelTests.swift

```diff
@@ -19,22 +19,30 @@
 import XCTest
 import WireCommonComponents
 @testable import Wire
+import WireSyncEngineSupport
 
 class CreateSecureGuestLinkViewModelTests: XCTestCase {
 
     // MARK: - Properties
 
-    var viewModel: CreateSecureGuestLinkViewModel!
+    var viewModel: CreateSecureConversationGuestLinkViewModel!
     var mockDelegate: MockCreatePasswordSecuredLinkViewModelDelegate!
     var textField: ValidatedTextField!
     var confirmPasswordField: ValidatedTextField!
+    var userSession: UserSessionMock!
```

Comment:
Why do you need the user session?

Reply:
Fixed! Done: https://github.com/wireapp/wire-ios/commit/ed6f88170f0cce8e92b3f41b81cb97417ea823f6